ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-stable-4.12-upgrade-from-stable-4.11.yaml

```diff
@@ -48,6 +48,13 @@ releases:
       version_bounds:
         lower: 4.11.0-0
         upper: 4.12.0-0
+  source:
```

Comment:
stable -> stable does not run any test suite, so we do not need the setting here.